packages/react-toolbar/src/components/Toolbar/Toolbar.types.ts

```diff
@@ -17,3 +18,11 @@ export type ToolbarProps = ComponentProps<ToolbarSlots> & ToolbarCommons;
  * State used in rendering Toolbar
  */
 export type ToolbarState = ComponentState<ToolbarSlots> & ToolbarCommons;
+
+export type ToolbarContextValue = {
+  size: ToolbarProps['size'];
```

Comment:
do you think that any other values will be in this context ? otherwise it can just be native context with primitive value

Reply:
at least until we see a reason to make it a context selector